Support disabling initial state

### Description
Previously when running a network that had a genesis state baked in you could override it and force the genesis state to be loaded from eth1 by setting `initial-state: ""` but that no longer works:

```
Teku failed to start.
java.util.concurrent.CompletionException: java.lang.IllegalStateException: Failed to load initial state
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:314)
	at java.base/java.util.concurrent.CompletableFuture.uniComposeStage(CompletableFuture.java:1190)
	at java.base/java.util.concurrent.CompletableFuture.thenCompose(CompletableFuture.java:2299)
	at tech.pegasys.teku.infrastructure.async.SafeFuture.thenCompose(SafeFuture.java:437)
	at tech.pegasys.teku.services.ServiceController.doStart(ServiceController.java:32)
	at tech.pegasys.teku.service.serviceutils.Service.start(Service.java:33)
	at tech.pegasys.teku.BeaconNode.start(BeaconNode.java:103)
	at tech.pegasys.teku.Teku.start(Teku.java:46)
	at tech.pegasys.teku.cli.BeaconNodeCommand.call(BeaconNodeCommand.java:280)
	at tech.pegasys.teku.cli.BeaconNodeCommand.call(BeaconNodeCommand.java:71)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1933)
	at picocli.CommandLine.access$1100(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2332)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2326)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2291)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2159)
	at picocli.CommandLine.execute(CommandLine.java:2058)
	at tech.pegasys.teku.cli.BeaconNodeCommand.parse(BeaconNodeCommand.java:229)
	at tech.pegasys.teku.Teku.main(Teku.java:32)
Caused by: java.lang.IllegalStateException: Failed to load initial state
	at tech.pegasys.teku.services.beaconchain.WeakSubjectivityInitializer.lambda$loadInitialAnchorPoint$0(WeakSubjectivityInitializer.java:54)
	at java.base/java.util.Optional.map(Optional.java:258)
	at tech.pegasys.teku.services.beaconchain.WeakSubjectivityInitializer.loadInitialAnchorPoint(WeakSubjectivityInitializer.java:44)
	at tech.pegasys.teku.services.beaconchain.BeaconChainController.initWeakSubjectivity(BeaconChainController.java:404)
	at tech.pegasys.teku.services.beaconchain.BeaconChainController.initialize(BeaconChainController.java:270)
	at tech.pegasys.teku.services.beaconchain.BeaconChainController.doStart(BeaconChainController.java:208)
	at tech.pegasys.teku.service.serviceutils.Service.start(Service.java:33)
	at tech.pegasys.teku.services.beaconchain.BeaconChainService.doStart(BeaconChainService.java:31)
	at tech.pegasys.teku.service.serviceutils.Service.start(Service.java:33)
	at tech.pegasys.teku.services.ServiceController.lambda$doStart$0(ServiceController.java:32)
	at java.base/java.util.concurrent.CompletableFuture.uniComposeStage(CompletableFuture.java:1183)
	... 17 more
Caused by: java.io.FileNotFoundException: Could not find 
	at tech.pegasys.teku.datastructures.util.ChainDataLoader.lambda$loadState$0(ChainDataLoader.java:28)
	at java.base/java.util.Optional.orElseThrow(Optional.java:401)
	at tech.pegasys.teku.datastructures.util.ChainDataLoader.loadState(ChainDataLoader.java:28)
	at tech.pegasys.teku.services.beaconchain.WeakSubjectivityInitializer.lambda$loadInitialAnchorPoint$0(WeakSubjectivityInitializer.java:48)
	... 27 more
Teku failed to start

To display full help:
teku [COMMAND] --help
```

--- a/teku/src/main/java/tech/pegasys/teku/cli/options/WeakSubjectivityOptions.java
+++ b/teku/src/main/java/tech/pegasys/teku/cli/options/WeakSubjectivityOptions.java
@@ -14,7 +14,6 @@
 package tech.pegasys.teku.cli.options;
 
 import java.util.Optional;
-
 import org.apache.commons.lang3.StringUtils;
 import picocli.CommandLine;
 import tech.pegasys.teku.cli.converter.CheckpointConverter;
@@ -55,7 +54,9 @@ public class WeakSubjectivityOptions {
     return builder.weakSubjectivity(
         wsBuilder -> {
           final Optional<String> initialStateResource =
-              Optional.ofNullable(weakSubjectivityState).or(networkDefinition::getInitialState).filter(StringUtils::isNotBlank);
+              Optional.ofNullable(weakSubjectivityState)
+                  .or(networkDefinition::getInitialState)
+                  .filter(StringUtils::isNotBlank);
           initialStateResource.ifPresent(wsBuilder::weakSubjectivityStateResource);
 
           if (weakSubjectivityCheckpoint != null) {

--- a/teku/src/main/java/tech/pegasys/teku/cli/options/WeakSubjectivityOptions.java
+++ b/teku/src/main/java/tech/pegasys/teku/cli/options/WeakSubjectivityOptions.java
@@ -14,6 +14,8 @@
 package tech.pegasys.teku.cli.options;
 
 import java.util.Optional;
+
+import org.apache.commons.lang3.StringUtils;
 import picocli.CommandLine;
 import tech.pegasys.teku.cli.converter.CheckpointConverter;
 import tech.pegasys.teku.config.TekuConfiguration;
@@ -53,7 +55,7 @@ public class WeakSubjectivityOptions {
     return builder.weakSubjectivity(
         wsBuilder -> {
           final Optional<String> initialStateResource =
-              Optional.ofNullable(weakSubjectivityState).or(networkDefinition::getInitialState);
+              Optional.ofNullable(weakSubjectivityState).or(networkDefinition::getInitialState).filter(StringUtils::isNotBlank);
           initialStateResource.ifPresent(wsBuilder::weakSubjectivityStateResource);
 
           if (weakSubjectivityCheckpoint != null) {

